graphics2d/src/test/java/de/rototor/pdfbox/graphics2d/MultiPageTest.java

```diff
@@ -43,13 +46,18 @@ public void testMultiPageJFreeChart() throws IOException {
 		parentDir.mkdirs();
 		File targetPDF = new File(parentDir, "multipage.pdf");
 		PDDocument document = new PDDocument();
+        HashMap<Image, PDImageXObject> seenImages = new HashMap();
+        Image pageLogo = ImageIO.read(
+                        MultiPageTest.class.getResourceAsStream("Rose-ProPhoto.jpg"));
 		for (int i = 0; i < 6; i++) {
 			PDPage page = new PDPage(PDRectangle.A4);
 			document.addPage(page);
 
 			PDPageContentStream contentStream = new PDPageContentStream(document, page);
 			PdfBoxGraphics2D pdfBoxGraphics2D = new PdfBoxGraphics2D(document, 800, 400);
+            pdfBoxGraphics2D.setRenderingHint(PdfBoxGraphics2D.RenderingHint.IMAGE_SHARING_MAP, seenImages);
```

Comment:
Using Rendering Hints here is not the correct way. We already have that functionality.